### DISCUSSION TITLE
Protein model parameters calculated from MW

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#1639] [Questions & Problems] [answered]
Dear all,

I have a question about the use of the MW in the calculations in the protein model.

I had a look at the structure of the model and according to my understanding the MW is only used to calculate two parameters in the protein model (considering a large molecule administered IV):
- **Radius**
- **Aqueous diffusion coefficient**

For this reason, two compounds, independently of their MW, should generate the same simulation if these two parameters are the same. I did this test and, the simulations are different (I created two compounds with different MW and mixed their respective parameters). 
Considering these results, it seems that the MW itself has an impact on the simulations. Does anybody have an explanation of this effect? I guess I am overlooking something.

I have attached the file with a dummy example I created.

Thank you in advance for your help!

Best,

Donato

[Dummy test calculation MW protein model.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/12894193/Dummy.test.calculation.MW.protein.model.zip)

**Comment by Christoph27** [comment, marked as answer]
Hi Donato,

you are right with, the same radius (and Kd(FcRn)) the proteins should behave the same (aqueous diffusion coefficient is relevant for particle dissolution, i.e. not part of the protein model). The difference in in your simulations stems from using the same mg-based dose. With the different molecular weights, the dose amount in µmol is different, thus the factor 2 difference in the simulated profiles (µmol/l).

Best, Christoph

**Reply by teutonicod** [reply]
Hi Christoph,

Thanks a lot! It is clear now!
